public/index.html

```diff
@@ -0,0 +1,155 @@
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Hello, Rachel</title>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+  <link rel="stylesheet" href="main.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+</head>
+<body>
+  <nav class="navbar navbar-default">
+    <div class="container-fluid">
+      <div class="navbar-header">
+        <a class="navbar-brand" style="background-color: rgb(253, 179, 108)" href="#">RPC Youth Group</a>
+      </div>
+      <ul class="nav navbar-nav">
+        <li class="active"><a style="background-color: rgb(252, 185, 115)" href="#">Home</a></li>
+        <li><a style="background-color: rgb(251, 191, 117)" href="#">Summer of 2014</a></li>
+        <li><a style="background-color: rgb(250, 197, 125)" href="#">Summer of 2015</a></li>
+        <li><a style="background-color: rgb(249, 203, 133)" href="#">Summer of 2016</a></li>
+        <li><a style="background-color: rgb(248, 209, 151)" href="#">Summer of 2017</a></li>
+        <li><a style="background-color: rgb(247, 215, 168)" href="#">Summer of 2018</a></li>
+        <li><a style="background-color: rgb(246, 215, 168)" href="#">Summer of 2019</a></li>
+        <li><a style="background-color: rgb(247, 222, 185)" href="#">Weekly Programming</a></li>
+
+      </ul>
+    </div>
+  </nav>
+  <div class="container-fluid bg-0">
+      <h1><img src="img/rpc.png" style="width: 3%; height: 3%; margin-top: 5px" class="img-circle" alt="RPC">Riverside Presbyterian Church Youth Group</h1>
+  </div>
+  <div class="container-fluid bg-1">
+
+    <h2>Who Are We?</h2>
+    <h3>We are a youth group that wants to make a difference! Our mission is to help those in need while spreading a message of inclusivity, love, service, and
+      kindness. Every summer, we embark on a youth service trip. Whether or not you are religious, there is a place for you here! Use the tabs above to read about our summer service projects, or scroll
+      through the pictures below!</h3>
+  </div>
+  <div class="container-fluid bg-2 text-center">
+    <div id="myCarousel" class="carousel slide" data-ride="carousel">
+      <!-- Indicators -->
+      <ol class="carousel-indicators">
+        <li data-target="#myCarousel" data-slide-to="0" class="active"></li>
+        <li data-target="#myCarousel" data-slide-to="1"></li>
+        <li data-target="#myCarousel" data-slide-to="2"></li>
+      </ol>
+
+      <!-- Wrapper for slides -->
+      <div class="carousel-inner">
+        <div class="item active">
+          <img src="img/door.jpg" alt="" style="width: 25%; height: 25%">
```

Comment:
It's kind of jarring that the carousel images change height as you scroll. You probably want the height to be fixed. If you do something like `height: 300px; width: auto;`, that will retain the aspect ratio while maintaining a fixed height between images.